docs/pipeline-components-and-applications/iglu/igluctl-2/index.md

```diff
@@ -19,13 +19,14 @@ Iglu provides a CLI application, called igluctl which allows you to perform most
     - `generate` - verify that schema is evolved correctly within the same major version (e.g. from `1-a-b` to `1-c-d`) for Redshift and Postgres warehouses. Generate DDLs and migrations from set of JSON Schemas. If the schema is not evolved correctly and backward incompatible data is sent within transformer's aggregation window, loading would fail for all events.
     - `push` - push set of JSON Schemas from static registry to full-featured (Scala Registry for example) one
     - `pull` - pull set of JSON Schemas from registry to local folder
-    - `verify-parquet` - verify that schema is evolved correctly within the same major version (e.g. from `1-a-b` to `1-c-d`) for parquet transformation (for loading into Databricks). It reports the breaking schema versions.
-    - `verify-redshift` - verify that schema is evolved correctly within the same major version (e.g. from `1-a-b` to `1-c-d`) for loading into Redshift. It reports the major schema versions within which schema evolution rules were broken.
     - `deploy` - run entire schema workflow using a config file. This could be used to chain multiple commands, i.e. `lint` followed by `push` and `s3cp`.
     - `s3cp` - copy JSONPaths or schemas to S3 bucket
 - `server` - work with an Iglu server
     - `keygen` - generate read and write API keys on Iglu Server
 - `table-check` - will check a given Redshift or Postgres tables against iglu server.
+- `verify` - work with schemas to check their evolution
```

Comment:
We need to say at the top of the doc which version of igluctl these instructions are for. Otherwise people might try these and get confused that it doesn’t work. I would also add:

```
`verify` (since 0.13.0) - ...
```

Reply:
would you also like to keep the previous bullets with `up to 0.13.0` note?